src/Strategy/DetermineConverterService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neusta\ConverterBundle\Strategy;
+
+/**
+ * @template S of object
+ * @template C of object
+ */
+interface DetermineConverterService
+{
+    public function determine(object $source, ?object $ctx = null): string;
+}
```

Comment:
What do you think of:

```suggestion
interface ConverterSelector
{
    public function selectConverter(object $source, ?object $ctx = null): string;
}
```